packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -96,6 +108,54 @@
         )
         currentVoteValue = participation?.answers[0] ?? null
     }
+
+    const handleTransferState = (state: TransferState): void => {
+        if (!state) return
```

Comment:
Can we make this more explicit? Returns should be communicated very clearly in code e.g.
```
if (!state) {
  return
}